### DISCUSSION TITLE
训练技巧

### DISCUSSION THREAD

**Discussion by zhangtingyun** [PaddlePaddle/PaddleOCR#13239] [Q&A]
有没有训练中文数据集比较好的模型 给我试试呀？  我的准确率一直上不去

**Comment by GreatV** [comment]
请参考 https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_ch/FAQ.md

**Reply by zhangtingyun** [reply]
没有看见pp-structurev2
版面分析相关的训练技巧欸

**Reply by SWHL** [reply]
版面分析模型就是一个目标检测或者图像分割任务，按照目标检测或图像分割来就行了。
360刚开源了一些版面分析模型，用的就是[YOLOv8](https://github.com/360AILAB-NLP/360LayoutAnalysis)
效果不好，一般是数据集太少了

**Reply by zhangtingyun** [reply]
但如果我训练集一直很少，一直训练，不应该会过于拟合吗？  那我们的较少数据集的正确率不应该会更高才对吗？  不去验证新的数据集的话

**Reply by SWHL** [reply]
可以做做实验呀。一直训一个数据集，然后在验证集上看看指标。起码证明框架没问题。
